pkg/sctx/sctx.go

```diff
@@ -88,6 +88,15 @@ func GetTargets(ctx context.Context) pss.Targets {
 	return targets
 }
 
+func padHexString(str string) string {
```

Comment:
actually this results in exactly the same thing :) disregard that.
as I said to lynn, we must specify the exact PO we'd like to mine out of the prefix in this case, there's no way around that, since a byte is still 2 words, and you're still creating a byte here

Reply:
Ok, closing the issue, and delete PR, if there is nothing to do.